runtime-modules/working-group/src/lib.rs

```diff
@@ -1298,7 +1298,7 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
         Ok(worker)
     }
 
```

Comment:
Add a comment for a public method, please.

Reply:
Done https://github.com/Joystream/joystream/pull/1273/commits/9d6bbb08706503ef06060959476baf0028b1fdf8